tutorials/exporting-stackdriver-elasticcloud/index.md

```diff
@@ -0,0 +1,337 @@
+[Services and APIs page](https://console.cloud.google.com/apis/dashboard) in the console before they can be used
+with `gcloud`. To perform the steps in this tutorial, enable the following APIs: 
+
+* Compute Engine
+* Cloud Pub/Sub
+* Identity and Access Management (IAM)
+* Google Stackdriver
+
+![Enable Cloud APIs](https://storage.googleapis.com/gcp-community/tutorials/exporting-stackdriver-elasticcloud/enable_apis.png)
+
+## Activate Google Cloud Shell
+
+The GCP Console provides an interactive shell that includes the `gcloud` command-line interface. At the top right corner of
+the page, click the **Activate Google Cloud Shell** button.
+
+![alt text](https://storage.googleapis.com/gcp-community/tutorials/exporting-stackdriver-elasticcloud/cloud_shell_icon.png)
+
+## Create a service account
+
+GCP [best practices](https://cloud.google.com/vpc/docs/firewalls#service-accounts-vs-tags) suggest using a 
+service account to configure security controls to a VM. A service account is useful for a VM to determine which other GCP 
+resources can be accessed by the VM and its applications, and which firewall rules should be applied to the VM.
+
+While credentials can be created to be used by a service account, this step is not necessary when the service account is
+attached to a VM running on Google Compute Engine. Google manages the keys, and applications can
+[retrieve the credentials securely](https://cloud.google.com/compute/docs/access/create-enable-service-accounts-for-instances#authenticating_applications_using_service_account_credentials)
+with the metadata service.
+
+1.  Create a service account to attach to the VM:
+
+        gcloud iam service-accounts create logstash \
+            --display-name="Logstash to Stackdriver"
+
+    **Expected response:**
+
+        Created service account [logstash].
+    
+1.  Provide permissions to access Cloud Pub/Sub using the `pubsub.editor`
+    [predefined IAM role](https://cloud.google.com/pubsub/docs/access-control#tbl_roles). _Editor_ permissions are required
+    for version 1.1.0 of Logstash Pub/Sub plugin. This appears to have been
+    [addressed](https://github.com/logstash-plugins/logstash-input-google_pubsub/blob/master/lib/logstash/inputs/google_pubsub.rb#L211-L214) in version 1.2.0, which means _subscriber_
+    permissions could be used.
+
+        gcloud projects add-iam-policy-binding scalesec-dev --member serviceAccount:logstash@scalesec-dev.iam.gserviceaccount.com --role roles/pubsub.editor
+
+## Create a Cloud Pub/Sub topic and subscription
+
+1.  Create a Cloud Pub/Sub topic where Stackdriver will send events to be picked up by Logstash:
+
+        gcloud pubsub topics create stackdriver-topic
+
+    **Expected response:**
+
+        Created topic [projects/scalesec-dev/topics/stackdriver-topic].
+
+    Version 1.1.0 of the Logstash plugin will automatically create a subscription.
```

Comment:
flagging this mention of version 1.1.0 as part of conversation in a later comment